src/main/resources/reference.conf

```diff
@@ -25,10 +25,11 @@ clusterResources {
   proxyDockerCompose = "proxy-docker-compose.yaml"
   welderDockerCompose = "welder-docker-compose.yaml"
   proxySiteConf = "cluster-site.conf"
-  googleSignInJs = "google_sign_in.js"
-  jupyterGooglePlugin = "google_plugin_jupyter.js"
+  googleSignInJs = "google_sign_in.js" //TODO make naming convention for nbextension js files
+  jupyterGooglePlugin = "google_plugin_jupyter.js" //TODO rename this because its name is misleading as to its actual purpose?
```

Comment:
+1 on renaming this, it's really an entrypoint for ALL js extensions, not just the google one

Reply:
done